turbiniactl

```diff
@@ -165,6 +165,9 @@ def PrintTaskStatus(instance, project, region, days=0, task_id=None,
       print '{0:s} request: {1:s} task: {2:s} {3:s} {4:s}: {5:s}'.format(
           task['last_update'], task['request_id'], task['id'], task['name'],
```

Comment:
Use the new logger here instead? Fine doing it in a later CL though.

Reply:
Yeah, here are lots of other print/logging changes that need to happen in this file.  I think I'll do them all at once as part of #104